README.md

```diff
@@ -1,75 +1,328 @@
+ ├─✔ collect-diff-results      cassandra-diff/collect-results                      workflow-adelphi-1-1334654991  16s                                     
+ ├─✔ gemini-oracle-sut         gemini-job/gemini-oracle-sut                        workflow-adelphi-1-3173132751  2m                                      
+ └─✔ start-results-server      results/results-server                              workflow-adelphi-1-1122498803  2s  
+ ```
 
-## Pre-requisites
+> Note that the `build-cassandra` and `build-cassandra-mgmt-api` steps of the workflow may not be executed with all runs of the workflow.
 
-In order to run Adelphi, you will need:
+## Remaining Resources
 
-- A running Kubernetes cluster (or use [k3d](https://k3d.io/) to run locally)
-- [Helm](https://helm.sh/) CLI (a k8s package manager)
-- Docker
-- [Argo](https://argoproj.github.io/) CLI (optional - for pretty printing progress output)
+At the completion of the workflow, a few resources are left running within the cluster:
 
-## Local Setup
+* Argo Server
+* Results Server
+* Cassandra Source Cluster
+* Cassandra Target Cluster
 
-If you don't have a k8s cluster available you can create one locally using k3d.  The instructions below assume the use of a local k3d cluster;
-if you're using some other existing cluster you can skip to step 4.
+These resources will remain available for inspection until the deployment is removed via `helm uninstall`
 
-1. [Install k3d](https://k3d.io/#installation) with the following command (or use one of the other options):
+## Retrieving Results
 
-    ```
-    curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
-    ```
+As testing completes within the workflow, results are made available via a webserver running within the cluster.
 
-2. Create a k8s cluster with 3 worker nodes that we will use to test Cassandra:
+### Port Forwarding
 
-    ```
-    k3d cluster create adelphi --servers 3 --wait
-    ```
+To access the Adelphi results web server, the `results-server` pod will need to be exposed from within the cluster.  When running locally, this can be done using `kubeclt port-forward` as shown:
 
-3. When that completes, set the KUBECONFIG environment variable to make sure you're working in the right context:
+```
+kubectl port-forward pod/results-server --namespace cass-operator :8080 
+```
 
-    ```
-    export KUBECONFIG="$(k3d kubeconfig get adelphi)"
-    ```
+This will enable port forwading on your local machine on an available port:
 
-4. If you don't have Helm installed you can [install](https://helm.sh/docs/intro/install/) it with the following command:
+```
+Forwarding from 127.0.0.1:64058 -> 8080
+Forwarding from [::1]:64058 -> 8080
+```
+
+You should now be able to lauch the Adelphi results web server from a browser on your local machine at http://localhost:64058/ to download the various results files generated by Adelphi.
```

Comment:
> You should now be able to lauch the Adelphi results web server from a browser on your local machine at http://localhost:64058/

As an user, this sentence would make me think the server would always be available at that port, but it was selected randomly at runtime. Maybe we can either set a port mapping `8080:8080` in the example or call out to check the port in the console output (`replace with the port you see in the output above`)

Reply:
Yeah, I went this route with the auto-selected port to avoid the problem of someone having a conflict on what we pick here and then being like, it doesn't work. The auto-selection will always work.  But, if you're not familiar, I can definitely see the confusion like you're mentioning.

Reply:
Also, it's probably worth mentioning here that you can download the results from the cluster with 

```
wget -r http://localhost:8080/
```

Reply:
I will definitely add that.  I didn't know until yesterday you mentioned it that it was possible, a good add for sure.